dspace-jspui/src/main/webapp/static/css/bootstrap/dspace-theme.css

```diff
@@ -105,4 +105,10 @@ span.batchimport-error-tab {
 
 span.batchimport-error-caused {
 	font-weight:bold;
+}
+
+.break-all{
+	-ms-word-break:   break-all;
+	word-break:       break-word;
```

Comment:
Css word-break must be set to break-all.
http://www.w3schools.com/cssref/css3_pr_word-break.asp